front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/HeroBanner/index.tsx

```diff
@@ -152,6 +189,18 @@ const EditCustomPageHeroBannerForm = ({
         <HelmetIntl title={messages.customPageMetaTitle} />
         <GenericHeroBannerForm
           onSave={handleSave}
+          onSaveAndEnable={
```

Comment:
This is an optional prop in the generic form since you can't turn on/off the section on the homepage